Detections/SecurityEvent/UserAccountEnabledDisabled_10m.yaml

```diff
@@ -25,22 +28,57 @@ query: |
   let timeframe = 1d;
   let spanoftime = 10m;
   let threshold = 0;
-  SecurityEvent
+  (union isfuzzy=true
+  (SecurityEvent
   | where TimeGenerated > ago(timeframe+spanoftime)
   // A user account was enabled
   | where EventID == 4722
   | where AccountType =~ "User"
   | where TargetAccount !hassuffix "$"
   | project EnableTime = TimeGenerated, EnableEventID = EventID, EnableActivity = Activity, Computer, UserPrincipalName, 
   AccountUsedToEnable = SubjectAccount, SIDofAccountUsedToEnable = SubjectUserSid, TargetAccount = tolower(TargetAccount), TargetSid
+  ),
+  (
+  WindowsEvent
+  | where TimeGenerated > ago(timeframe+spanoftime)
+  // A user account was enabled
+  | where EventID == 4722 and EventData has "User"
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend SubjectAccount = strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(SubjectAccount endswith "$" or SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(SubjectUserSid), "", "User")
+  | where AccountType =~ "User"
+  | extend TargetAccount = strcat(EventData.TargetDomainName,"\\", EventData.TargetUserName)
+  | where TargetAccount !hassuffix "$"
```

Comment:
This is a bug in the original. Should be !endswith. Correct also in the original.